docs/cloud-common.md

```diff
@@ -1,6 +1,30 @@
-# Set up Ark with your cloud provider
+# Set up Ark on your platform
 
-To run Ark with your cloud provider, you specify provider-specific settings for the Ark server. In version 0.7.0 and later, you can run Ark in any namespace, which requires additional customization. See [Run in custom namespace][3].
+You can run Ark with a cloud provider or on-premises. For detailed information about the platforms that Ark supports, see [Compatible Storage Providers][99].
+
+In version 0.7.0 and later, you can run Ark in any namespace, which requires additional customization. See [Run in custom namespace][3].
+
+In version 0.9.0 and later, you can use Ark's integration with restic, which requires additional setup. See [restic instructions][20].
+
+## Set up Ark on-premises
+
+You can run Ark in an on-premises cluster in different ways depending on your requirements.
```

Comment:
Note for self: I think I want to just add some language here saying that if you're on-prem, you'll need to mix and match object and block storage options, along with possibly restic, and add links to the relevant pages/sections (support matrix, restic, ...).  No need to duplicate existing content here.